packages/suite/src/hooks/wallet/useSendFormImport.ts

```diff
@@ -41,12 +51,22 @@ export const useSendFormImport = ({ network, tokens, localCurrencyOption, fiatRa
 
                 if (currency === network.symbol) {
                     // csv amount in crypto currency
-                    output.amount = item.amount || '';
+                    const cryptoValue = item.amount || '';
+                    if (!isInteger(cryptoValue) && shouldSendInSats) {
+                        // try to convert to satoshis
+                        output.amount = amountToSatoshi(cryptoValue, network.decimals);
+                    } else {
+                        output.amount = cryptoValue;
+                    }
+
                     // calculate Fiat from Amount
                     if (fiatRates && fiatRates.current) {
+                        const fiatAmount = shouldSendInSats
+                            ? formatAmount(output.amount, network.decimals)
+                            : output.amount;
                         output.fiat =
                             toFiatCurrency(
-                                output.amount,
+                                fiatAmount,
```

Comment:
I think the naming is off here (the value represents crypto amount) and perhaps it can be simplified:

Reply:
nope, it represents fiat amount

Reply:
i was confused by `output.fiat`, `fiatAmount` is actually `cyptoAmountInBitcoinUnits` (exaplined in comment below)

i've slightly renamed it https://github.com/trezor/trezor-suite/pull/10327/commits/aaf41b6535a8c186c09fcb3be01521d0b44250e4